include/darshan-pnetcdf-log-format.h

```diff
@@ -7,45 +7,331 @@
 #ifndef __DARSHAN_PNETCDF_LOG_FORMAT_H
 #define __DARSHAN_PNETCDF_LOG_FORMAT_H
 
-/* current PNETCDF log format version */
-#define DARSHAN_PNETCDF_VER 2
-
-#define PNETCDF_COUNTERS \
-    /* count of PNETCDF independent opens */\
-    X(PNETCDF_INDEP_OPENS) \
-    /* count of PNETCDF collective opens */\
-    X(PNETCDF_COLL_OPENS) \
+/* current PnetCDF log format version */
+#define DARSHAN_PNETCDF_FILE_VER 2
+#define DARSHAN_PNETCDF_VAR_VER 1
+
+#define PNETCDF_VAR_MAX_NDIMS 5
+
+/* counters for file level activities */
+#define PNETCDF_FILE_COUNTERS \
+    /* count of file creates */\
+    X(PNETCDF_CREATES) \
```

Comment:
For uniformity, could we prefix all of these file module counters with `PNETCDF_FILE` similar to how the variable module counters are all prefixed with `PNETCDF_VAR`? I think it's more clear that way.

Reply:
OK with me.

Reply:
Resolved via [e1d6ca6](https://github.com/darshan-hpc/darshan/pull/456/commits/e1d6ca67029ecba3088150568bee25fea8f2ba8a)

This commit is just to get things building, I haven't updated the darshan-util doc descriptions of counters, since there are still some more general changes to make to counters first.